qdrant-landing/content/documentation/guides/installation.md

```diff
@@ -6,21 +6,74 @@ aliases:
   - ../installation
 ---
 
-# Installation options
+## Installation requirements
 
-## Docker
+### CPU and memory
 
-The easiest way to start using Qdrant is to run it from a ready-made Docker image.
-The latest versions are always available on [DockerHub](https://hub.docker.com/r/qdrant/qdrant/tags?page=1&ordering=last_updated).
+The amount of CPU and memory that Qdrant needs depends on the amount of vectors, their dimensions, the additional payload data that you store, and their indexes, as well as storage, replication and quantization configuration.
+
+Our [Cloud Pricing Calculator](https://cloud.qdrant.io/calculator) can give you a very rough approximation of the necessary resources to store the vectors without any payload or index data.
+
+### Storage
+
+For persistent storage, Qdrant requires a block storage device with a proper file system such es ext4. This is especially important, if you mount an external volume inside the Qdrant container or if you use a Kubernetes Persistent Volume.
+Qdrant won't work on a file storage device, such as NFS, or an object storage such as S3.
+
+The amount of storage depends on the size of your data.
+
+If you offload vectors to disk, you also have to ensure that your disk storage is performant. You should at least use SSDs or NVMEs. HDDs are not recommended.
+
+### Networking
+
+Each Qdrant instance opens 3 ports:
+
+* `6333` - For the HTTP API, including the health and metric endpoints
+* `6334` - For the gRPC API
+* `6335` - For internal replication communication
+
+All Qdrant instances in a cluster must be able to communicate with each other over these ports.
+
+Each Qdrant instance must also allow incoming connections to ports `6333` and `6334` from the clients that will be using Qdrant.
+
+## Installation options
```

Comment:
I think it's better to have the production setups first. Otherwise people will only read half of the page, use the first example they see and then run a dev setup in production.